src/main/java/com/google/api/gax/core/RetrySettings.java

```diff
@@ -72,44 +69,45 @@
 public abstract class RetrySettings {
 
   /**
-   * See the class documentation of {@link RetrySettings} for a description
-   * of what this value does.
+   * The TotalTimeout parameter has ultimate control over how long the logic should keep trying the
+   * remote call until it gives up completely. The higher the total timeout, the more retries can be
+   * attempted.
    */
   public abstract Duration getTotalTimeout();
 
   /**
-   * See the class documentation of {@link RetrySettings} for a description
-   * of what this value does.
+   * The InitialRetryDelay parameter controls the delay before the first retry. Subsequent retries
+   * will use this value adjusted according to the RetryDelayMultiplier.
    */
   public abstract Duration getInitialRetryDelay();
 
   /**
-   * See the class documentation of {@link RetrySettings} for a description
-   * of what this value does.
+   * The RetryDelayMultiplier controls the change in retry delay. The retry delay of the previous
+   * call is multiplied by the RetryDelayMultiplier to calculate the retry delay for the next call.
    */
   public abstract double getRetryDelayMultiplier();
 
   /**
-   * See the class documentation of {@link RetrySettings} for a description
-   * of what this value does.
+   * The MaxRetryDelay puts a limit on how much the retry delay can be increased by the
+   * RetryDelayMultiplier.
```

Comment:
-> "puts a limit on the value of the retry delay, so that the RetryDelayMultiplier can't increase the retry delay higher than this amount."